terraform/infrastructure/03-main.tf

```diff
@@ -0,0 +1,98 @@
+﻿module "network" {
+  source = "./modules/01-network"
```

Comment:
is there any particular reason to put numbers in the module names? 
It probably makes sense to just call them `network`, `static-website`, `database`, etc. since they're all independent and their dependencies are self-contained in their variables.